### PR TITLE
[devel] Update Feet FT IMU configurations in inertial wrapper and hardware

### PR DESCRIPTION
**Pull request by prashanthr05**
This PR changes the following,
- Updates the feet FT IMU names to be coherent with the sensor names generated in the URDF through the PR https://github.com/robotology/icub-model-generator/pull/122. Consequently fixes https://github.com/robotology/robots-configuration/issues/125
- Adds FT IMUs orientation sensors and magnetometer sensors to the right leg and left leg inertial MAS remapper

For more details on the duplication of PR https://github.com/robotology/robots-configuration/pull/137, please see the comment https://github.com/robotology/robots-configuration/pull/137#issuecomment-563909755

### CODE REVIEW

**Comment by traversaro** [comment]
There seems to be much more changes then just the one listed in the PR. Can you cherry-pick just the last two commits or it is not possible?

**Comment by prashanthr05** [comment]
Uhmm, all the changes from devel_iCubGenova04 have been carried over. 
In my opinion, the last 4 commits are important for the feet FT IMUs. But I do not fully understand the implications of the complete history. If somewhere in some commit, a particular change relating to these IMUs were made?

What do you suggest I do?

**Comment by prashanthr05** [comment]
@nunoguedelha @GiulioRomualdi Please feel free to close this PR, if you think it is not relevant anymore. Thank you.

**Comment by marcoaccame** [comment]
Hi @nunoguedelha @GiulioRomualdi @prashanthr05, I read the following:

> @nunoguedelha @GiulioRomualdi Please feel free to close this PR, if you think it is not relevant anymore. Thank you.


**Can I close the PR??**

pls let me know. if you still need to push any changes, we can understand the best way forward with a brief meeting f2f.

**Comment by traversaro** [comment]
Friendly ping @GiulioRomualdi @prashanthr05 @nunoguedelha @S-Dafarra

**Comment by GiulioRomualdi** [comment]
Hi @marcoaccame and @traversaro. I checked the status of the code. This PR was opened by @prashanthr05 for enabling the usage of the feet IMU. Namely https://github.com/robotology/robots-configuration/pull/141/commits/efcd8a8cb9d65bd19783bc8437d6ab7ac6ba4344 https://github.com/robotology/robots-configuration/pull/141/commits/f8387e731273a989fa0b6121e2dffa5c9b2e2ff2 https://github.com/robotology/robots-configuration/pull/141/commits/85cbef0f05a5f6335904ecfb6327c898232cc341

However, these commits were already merged into `cleanup/2019.12.feat-01`  Thanks to this PR: https://github.com/robotology/robots-configuration/pull/119
image
image

And `cleanup/2019.12.feat-01` has been merged in master with this pr: https://github.com/robotology/robots-configuration/pull/143

In conclusion, we can close this PR

**Comment by pattacini** [comment]
Thanks @GiulioRomualdi for the investigation 👍 
Closing then.
